src/components/activity/ActivityAnchorForm.svelte

```diff
@@ -180,6 +189,10 @@
             class="secondary anchor-boundary"
             class:selected={!isAnchoredToStart}
             class:disabled
+            use:permissionHandler={{
```

Comment:
Could you add this custom action to the tags input in the directive form? Should support it.